WordPress/src/main/java/org/wordpress/android/ui/prefs/DotComSiteSettings.java

```diff
@@ -91,6 +94,10 @@
     public void saveSettings() {
         super.saveSettings();
 
+        if (mSite.isJetpackConnected()) {
+            saveJetpackSettings();
```

Comment:
I believe this could result in similar issues mentioned below for fetching jetpack settings due to `mRemoteSettings.copyFrom(mSettings);` call. I am not so sure saving/fetching to/from multiple endpoints and changing the same model for it. Is there a better way to handle this?